GoogleUtilities/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+- `GULAppDelegateSwizzler`: support of remote notification methods. (#2698)
```

Comment:
Is this file intended to be in this PR?

Reply:
Well, probably, the changelog changes should have been better be in a different PR, but I would prefer to merge the PR to avoid waiting for another build to finish.